lib/voorhees/json_api.ex

```diff
@@ -19,6 +19,33 @@ defmodule Voorhees.JSONApi do
     actual
   end
 
+  def assert_schema_contains(%{"data" => resource} = actual, expected) when is_map(resource) do
+    assert_schema_contains(put_in(actual["data"], [resource]), expected)
+  end
+  def assert_schema_contains(%{"data" => resources} = actual, expected) when is_list(resources) do
+    actual = Enum.map(resources ++ actual["included"] || [], fn(resource) -> _stringify_keys(resource) end)
+    _assert_schema_contains(actual, _stringify_keys(expected))
+  end
+
+  defp _assert_schema_contains(actual, expected) do
+    actual_types = Enum.map(actual, fn(resource) -> resource["type"] end)
+    expected_types = Map.keys(expected)
+
+    assert length(expected_types -- actual_types) == 0,
```

Comment:
yes but this is asserting the difference between the expected_types and the actual_types:

```elixir
expected_types = [1,2]
actual_types = [2,3]
expected_types -- actual_types == [1]

expected_types = [2,3,4]
actual_types = [1,2,3,4,5,6]
expected_types -- actual_types == []
```

Because Elixir/Erlang doesn't have an `intersection` operator on Lists (it does on Sets but I was not sure how to use it with a List) this seems to be the best solution.